src/layouts/Header/Header.tsx

```diff
@@ -14,7 +18,12 @@ function Header() {
     <div className="container flex h-14 w-full items-center">
       <div className="flex-1">Navigation</div>
```

Comment:
Probably just me, but it's a little weird we have the word Navigation here lol Maybe something we can double check with design if they're keen on keeping?

Reply:
Ignore this, I see you're building up the component in pieces 🙈

Reply:
Navigation 🗺️